plugin/kubernetes/setup.go

```diff
@@ -229,6 +229,18 @@ func ParseStanza(c *caddy.Controller) (*Kubernetes, error) {
 				continue
 			}
 			return nil, c.ArgErr()
+		case "namespace_labels":
```

Comment:
I am contemplating whether or not the usage of `namespace_labels` requires that `namespaces` be `nil`. I can't think of a good reason why someone would want to use both `namespaces` and `namespace_labels` in conjunction with each other.

Reply:
Went ahead and added this logic in a separate commit.